db/queries/assessment_records/queries.py

```diff
@@ -576,8 +576,8 @@ def get_assessment_records_score_data_by_round_id(round_id, selected_fields=None
     default_fields = [
         "Application ID",
         "Short ID",
-        "Score Subcriteria",
         "Score",
+        "Score Subcriteria",
```

Comment:
I wonder if we can extend the `test_output_tracker_data()` to write a failing test that can then guard against regression on the order of the scoring columns based on if its scored or not

I'm happy to pair on this if helpful but maybe it could be something along the lines of:
- call the export data method with no scores in the database 
- add scores to the database
- call the export data method
- assert that the `.keys()` of the dictionaries in both returned lists start with the same thing

Strangely it looks like these tests actually use the database rather than fixtures so there may be some sharp edges but hopefully theres something to go on.

I don't think that this needs to block this PR, especially if appeasing the tests takes longer than reasoning about the change!